atom/browser/api/atom_api_web_contents.h

```diff
@@ -92,6 +92,8 @@ class WebContents : public mate::TrackableObject<WebContents>,
   void GoBack();
   void GoForward();
   void GoToOffset(int offset);
+  const std::string& GetWebRTCIPHandlingPolicy() const;
```

Comment:
Looks like our other getter APIs like `GetUserAgent` and `GetTitle` don't return references so I think it is fine to just return a `std::string` here for consistency.